drivers/virtualbox/virtualbox.go

```diff
@@ -58,11 +59,24 @@ func init() {
 	})
 }
 
+func GetCPUCores() int {
+	cpus := runtime.NumCPU()
+	if cpus > 32 {
+		cpus = 32
+	}
+	return cpus
+}
+
 // RegisterCreateFlags registers the flags this driver adds to
 // "docker hosts create"
 func GetCreateFlags() []cli.Flag {
 	return []cli.Flag{
 		cli.IntFlag{
+			Name:  "virtualbox-processors",
+			Usage: "Number of virtual CPUs cores for host",
```

Comment:
Should probably mention that it defaults to the number of cores on the host. The default might seem to make it obvious, but it's easy to miss the correlation ;)